content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -38,6 +38,35 @@ that provides the expected symbols.
 
 <!-- steps -->
 
+## Check your OS kernel version
+
```

Comment:
on this line we need to add `{{% thirdparty-content %}}` if we link to external pages outside of k8.io

Reply:
Fixed in https://github.com/kubernetes/website/pull/50375/commits/66b032bdbdd12d84aee7ed3e95bbe4586147f230